src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -58,17 +60,18 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
 
             new CompletionController(textViewAdapter, textView, CompletionBroker);
 
-
             _dependencies = Dependencies.FromConfigFile(doc.FilePath);
             _manifest = Manifest.FromFileAsync(doc.FilePath, _dependencies, CancellationToken.None).Result;
             _manifestPath = doc.FilePath;
             _project = VsHelpers.GetDTEProjectFromConfig(_manifestPath);
+            IEnumerable<ILibraryOperationResult> results = await LibrariesValidator.GetManifestErrorsAsync(_manifest, _dependencies, CancellationToken.None).ConfigureAwait(false);
 
```

Comment:
This work should be queued to run after we've finished initializing the text view, and it can be run async (e.g. with JTF)